docs/sources/alerting/notifications.md

```diff
@@ -230,3 +230,14 @@ Notification services which need public image access are marked as 'external onl
 
 All alert notifications contain a link back to the triggered alert in the Grafana instance.
 This URL is based on the [domain]({{< relref "../administration/configuration/#domain" >}}) setting in Grafana.
+
+## Templating
```

Comment:
This should be a task and explain how to use variables in alert notifications, why you would want to, and some examples. Perhaps the ones you showed in the Thursday meeting?

Reply:
Hopefully, the current state is more in line with what you're thinking @oddlittlebird. Let me know if there's anything I should change 😄